tests/test_implementations/test_rustsim_implementation.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+from rustsim import get_intersection, jaccard_similarity, mrca_and_score
+
+from oaklib.datamodels.vocabulary import IS_A, PART_OF
+from oaklib.implementations.sqldb.sql_implementation import SqlImplementation
+from oaklib.resource import OntologyResource
+from oaklib.selector import get_adapter
+from tests import ENDOMEMBRANE_SYSTEM, INPUT_DIR, VACUOLE
+from tests.test_implementations import ComplianceTester
+
+DB = INPUT_DIR.joinpath("go-nucleus.db")
+
+
+class TestRustSimImplementation(unittest.TestCase):
+    """Implementation tests for Rust-based semantic similarity."""
+
+    def setUp(self) -> None:
+        """Set up"""
+        # Calling get_implementation_from_shorthand() alone fails on Windows
+        try:
+            oi = get_adapter(f"rustsim:sqlite:///{str(DB)}")
```

Comment:
```python
>>> from pathlib import Path
>>> f"rustsim:sqlite:///{Path(DB).as_posix()}"
'rustsim:sqlite:///C:/this.db'
```

Reply:
Thanks for the suggestion @dalito ! Unfortunately, that didn't work. The problem lies [here](https://github.com/INCATools/ontology-access-kit/blob/main/src/oaklib/selector.py#L184-L188)

```
        if ":" in descriptor and not os.path.exists(descriptor):
            toks = descriptor.split(":")
            scheme = toks[0]
            resource.scheme = scheme
            rest = ":".join(toks[1:])
```